<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PasswordLineEdit" name="RootPassword_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>391</width>
     <height>33</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="PrimaryPushButton" name="StartTheConf_config">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>153</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start </string>
   </property>
  </widget>
  <widget class="PushButton" name="CancelTheConf_config">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>102</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="StrongBodyLabel" name="StrongBodyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the machine's root user and password</string>
   </property>
  </widget>
  <widget class="BodyLabel" name="BodyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please be aware that the password you're entering in this form </string>
   </property>
  </widget>
  <widget class="BodyLabel" name="BodyLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>691</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>will be utilized solely for configuring the machine and will not be </string>
   </property>
  </widget>
  <widget class="BodyLabel" name="BodyLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>stored anywhere afterward.</string>
   </property>
  </widget>
  <widget class="LineEdit" name="RootUser_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>33</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Root user</string>
   </property>
  </widget>
  <widget class="PasswordLineEdit" name="MasterPassword_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>391</width>
     <height>33</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Master Password</string>
   </property>
  </widget>
  <widget class="ProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>391</width>
     <height>4</height>
    </rect>
   </property>
  </widget>
  <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
